docs/catalog/web/deprecate-gifs.md

```diff
@@ -0,0 +1,44 @@
+---
+version: 1.0
+submitted_by: markus-ntt-seidl
+published_date: TBD
+category: 
```

Comment:
Still needs to have `web` in the category field for properly pathing in the catalog on the site (it is in the tag list already)